modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-size.adoc
+
+[id="configuring-{prod-id-short}-workspace-pvc-size"]
+= Configuring the storage size in {prod-short}
```

Comment:
Not sure, but it might be a bit redundant to say "in Che/DevSpaces" since this is the Che/DevSpaces docs.

Reply:
The `id` and the `=` title have to be the same (except for the lowercase dashed syntax).

Reply:
Correct, we can avoid Che/DevSpaces when it's self-explanatory to our readers.